### DISCUSSION TITLE
Set LibreChat PWA to use svg icon on Windows taskbar

### DISCUSSION THREAD

**Discussion by kneelesh48** [danny-avila/LibreChat#4094] [Feature Requests & Suggestions]
### What happened?

LibreChat PWA icon on Windows taskbar has white area around it as it's using a JPEG.

I'd like it to use a PNG and make the white area transparent

![image](https://github.com/user-attachments/assets/d8a096b5-ed92-4024-a1c4-a58ee23a9a28)
This is what it looks like

### Steps to Reproduce

1. Open Librechat it browser
2. Install it as an app in Windows

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MNLierman** [comment]
So damn ugly!

**Comment by MNLierman** [comment]
Why you thumbs down what is absolutely the truth? It took me 8 minutes and I created this. Anyone can use this and if you're using Edge, go to Menu 🡒 Apps 🡒 Install 🡒 Change icon, select this one I'm attaching.  The dev could have done this, but he chooses not to. 

The result:
![image](https://github.com/user-attachments/assets/af5ea220-8e9d-47c6-88d0-4e733327c0f0)

The icon:
![LC](https://github.com/user-attachments/assets/e5747678-8d98-4d9a-b8c2-3f50c7c279a5)

**Reply by danny-avila** [reply]
This is an open source project, all you need to do is fork, create a branch, and submit a pull request, rather than complain.

**Reply by MNLierman** [reply]
I didn't complain, I provided a solution. I don't have time to find where the icon is, fork, recompile, submit PR, and wait for it to maybe be accepted, and in the meantime I have fragmented code. Most project maintainers do not accept pull requests. It's why there is the following joke going around:
“If you're camping/hiking/in the forest, and you come across a bear, ask him to accept your pull request and he go off and pretend you don't exist.”

**Reply by danny-avila** [reply]
I appreciate you sharing that perspective and understand the frustration with PR processes. Just to note - this repo actually has a strong track record of accepting contributions, with over 700 merged PRs from our community. Specific suggestions like 'the icon should be located at X path' or 'the code needs Y change' are more helpful than general assumptions about maintainers not accepting PRs. I'm always seeking to improve this repo and I'm very much aware I can't do it alone.
